torch/csrc/jit/script/compiler.cpp

```diff
@@ -1328,6 +1431,39 @@ struct to_ir {
     return n;
   }
 
+  Node* emitNodeFromSchema(
```

Comment:
There is a lot of overlap between this code and `emitBuiltinCall`. Is it possible to just call, e.g. `emitBuiltinCall("add", ...`.

The problem is that this code duplication is very fragile. For instance, liftConstantAttributes is not being called in this pathway but is being called in the builtin pathway.

Reply:
A follow up advantage is that some of the logic for '+' and '-' will be handled naturally by `emitBuiltinCall`.